mona_sdk/client.py

```diff
@@ -1089,22 +1139,22 @@ def _app_server_request(self, endpoint_name, data=None):
                 headers=get_basic_auth_header(
                     self.api_key, self.should_use_authentication
                 ),
-                # Remove keys with UNPROVIDED_FIELD values to avoid overriding the
-                # default value on the endpoint itself.
-                json=remove_items_by_value(data, UNPROVIDED_VALUE) if data else {},
+                # Remove keys with UNPROVIDED_FIELD values to avoid overriding
+                # the default value on the endpoint itself.
+                json=(remove_items_by_value(data, UNPROVIDED_VALUE) if data else {}),
             )
             json_response = app_server_response.json()
             if not app_server_response.ok:
-                return (
-                    self._handle_service_error(json_response["response_data"])
-                    if (
-                        app_server_response.status_code == 400
-                        and json_response
-                        and "response_data" in json_response
-                    )
-                    else self._handle_service_error(SERVICE_ERROR_MESSAGE)
+                kwargs = {
```

Comment:
Feels a bit weird to create this dict just to destruct it in one place. Why not just putting it there?